gremlin-dotnet/glv/Gremlin.Net.csproj.template

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Build">
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
I think there's no need to add `netcoreapp3.0` as a target for the core project, given that `netcoreapp3.0` implements `netstandard2.0`.

Reply:
I wasn't sure here whether it makes sense or not. I thought that doing it like this is necessary to prevent the SDK from installing the NuGet package for `System.Text.Json` on .NET Core >= 3.0 as it's already included there. Or is that already detected automatically?

Reply:
It will be resolved automatically for `netcoreapp3.0` users.

Reply:
Good to know. Then I'll remove this target again.